src/Book.java

```diff
@@ -23,4 +27,27 @@ public int getPublicationYear() {
     public void setPublicationYear(int year) {
         this.publicationYear = year;
```

Comment:
```suggestion
        publicationYear = year;
```
можно не использовать this, т.к. названия поля и параметра разные